src/navigation/Nav.jsx

```diff
@@ -252,6 +253,15 @@ export class Nav extends React.Component {
 				),
 			},
 			media.isAtMediumUp && !self.is_pro_admin && createMeetupLink,
+			// If you want to use pill elsewhere, consider making a Pill
+			// component that takes a color prop.
+			media.isAtMediumUp && {
+				shrink: true,
+				linkTo: experiences.link,
```

Comment:
What happens if `experiences` is undefined? Do you need to update each usage of `Nav` in pro-web and mup-web to guarantee the existence of this?

Reply:
We had added a null check but hadn't committed it. Committed and pushed.